website/docs/r/aviatrix_gateway.html.markdown

```diff
@@ -215,6 +215,9 @@ The following arguments are supported:
 * `enable_monitor_gateway_subnets` - (Optional) If set to true, the [Monitor Gateway Subnets](https://docs.aviatrix.com/HowTos/gateway.html#monitor-gateway-subnet) feature is enabled. Default value is false. Available as of provider version R2.18+.
 * `monitor_exclude_list` - (Optional) A list of monitored instance IDs separated by comma when Monitor Gateway Subnets feature is enabled. Default value is "". Available as of provider version R2.18+.
 
+### FQDN Gateway
+* ` fqdn_lan_cidr` - (Optional) If `fqdn_lan_cidr` is set, the FQDN gateway will be created with an additional LAN interface using the provided cidr. This attribute is required when enabling FQDN gateway firenet in Azure. Available in provider version R2.17+
```

Comment:
Should say `R2.17.1+`. same with the other changes to the docs, all should say R2.17.1+